src/components/card/CardSidebarTabAttachments.vue

```diff
@@ -22,30 +22,254 @@
 
 <template>
 	<div>
-		{{ card.attachments }}
 		<button class="icon-upload" @click="clickAddNewAttachmment()">
 			{{ t('deck', 'Upload attachment') }}
 		</button>
+		<input ref="localAttachments"
+			type="file"
+			style="display: none;"
+			@change="onLocalAttachmentSelected">
+		<div class="attachment-list">
+			<ul>
+				<li v-for="attachment in attachments"
+					:key="attachment.id"
+					class="attachment"
+					style="display: flex;">
+					<a class="fileicon" :style="mimetypeForAttachment(attachment)" :href="attachmentUrl(attachment)" />
+					<div class="details">
+						<a :href="attachmentUrl(attachment)" target="_blank">
+							<div class="filename">
+								<span class="basename">{{ attachment.data }}</span>
+							</div>
+							<span class="filesize">{{ formattedFileSize(attachment.extendedData.filesize) }}</span>
+							<span class="filedate">{{ relativeDate(attachment.createdAt*1000) }}</span>
+							<span class="filedate">{{ attachment.createdBy }}</span>
+						</a>
+					</div>
+					<Actions>
+						<ActionButton v-if="attachment.deletedAt === 0" icon="icon-delete" @click="deleteAttachment(attachment)">
+							{{ t('deck', 'Delete Attachment') }}
+						</ActionButton>
+
+						<ActionButton v-else icon="icon-history" @click="restoreAttachment(attachment)">
+							{{ t('deck', 'Restore Attachment') }}
+						</ActionButton>
+					</Actions>
+				</li>
+			</ul>
+		</div>
+
+		<Modal v-if="modalShow" title="File already exists" @close="modalShow=false">
+			<div class="modal__content">
+				<h2>{{ t('deck', 'File already exists') }}</h2>
+				<p>
+					{{ t('deck', 'A file with the name') }}
```

Comment:
Constructing language strings like this doesn't work in all languages. Please use a placeholder like 
```suggestion
					{{ t('deck', 'A file with the name {filename} already exists.', {filename: file.name}) }}
```